portal-api/src/v1/api-response-types/utilisation-report-pending-corrections-response-body.ts

```diff
@@ -0,0 +1,18 @@
+import { IsoDateTimeStamp, ReportPeriod } from '@ukef/dtfs2-common';
+
+type PendingCorrection = {
+  correctionId: number;
+  facilityId: string;
+  exporter: string;
+  additionalInfo: string;
+};
+
+export type UtilisationReportPendingCorrectionsResponseBody =
```

Comment:
could this also be in libs/common?

Reply:
We haven't been doing this for types which differ in central api because the response body is quite tied to the service and we don't want people getting confused about which services they belong in, e.g. this type is different in central-api because in central-api the date uploaded is a `Date` but in the apis consuming it it will have been stringified into an `IsoDateTimeStamp` so the types are different in different services.